mariadb.spec

```diff
@@ -1310,6 +1300,57 @@ fi
 %{_libdir}/mysql/plugin/ha_connect.so
 %endif
 
+%files server-utils
+%if %{with client}
+%{_bindir}/mysql_find_rows
+%{_bindir}/mysqlaccess
+%{_mandir}/man1/mysqlaccess.1*
+%{_mandir}/man1/mysql_find_rows.1*
+%endif
+#Perl utilities
+%{_bindir}/mysql_convert_table_format
+%{_bindir}/mysql_fix_extensions
+%{_bindir}/mysql_setpermission
+%{_bindir}/mysql_zap
+%{_bindir}/mysqldumpslow
+%{_bindir}/mysqld_multi
+%{_bindir}/mysqlhotcopy
+%{_mandir}/man1/mysql_convert_table_format.1*
+%{_mandir}/man1/mysql_fix_extensions.1*
+%{_mandir}/man1/mysql_zap.1*
+%{_mandir}/man1/mysqldumpslow.1*
+%{_mandir}/man1/mysqld_multi.1*
+%{_mandir}/man1/mysqlhotcopy.1*
+%{_mandir}/man1/mysql_setpermission.1*
+#Utilities that can be used remotely
+%{_bindir}/mysql_upgrade
+%{_bindir}/mysqltest
+%{_bindir}/perror
+%{_mandir}/man1/mysql_upgrade.1*
+%{_mandir}/man1/mysqltest.1*
+%{_mandir}/man1/perror.1*
+#Utilites that are not necessary, but should not be used remotely (because of huge data flow)
+%{_bindir}/mysqlcheck
+%{_bindir}/mysqldump
+%{_bindir}/mysqlimport
+%{_bindir}/mysqlslap
+%{_mandir}/man1/mysqlcheck.1*
+%{_mandir}/man1/mysqldump.1*
+%{_mandir}/man1/mysqlimport.1*
+%{_mandir}/man1/mysqlslap.1*
+#Other utilities
+%{_bindir}/mysqld_safe_helper
```

Comment:
here's mysqld_safe_helper, this is needed by mysqld_safe in 10.1.30, not in 10.1.20, so this is new.

Reply:
ok, that was pulled in Fedora 10.1.30 spec via
Recommends:       %{name}-server-utils%{?_isa} = %{sameevr}
which I commented out in RDO/EL7 build.
I'll add it as Requires:

Reply:
@zzzeek mariadb-10.1.30-2.1.el7 http://cbs.centos.org/koji/buildinfo?buildID=21184 with Recommends: -> Requires: which should fix this issue.
@dciabrin @zzzeek There is one more question for you in https://review.rdoproject.org/r/11342 about oqgraph-engine - do we need it or it can be disabled?

Reply:
oqgraph seems to be some mariadb specific thing and was actually disabled in mariadb 5.5, so it's safe to say nothing in openstack is using this.